types/Rule.ts

```diff
@@ -91,3 +94,33 @@ export interface DeleteRuleParams {
 }
 
 export interface DeleteRuleResponse {}
+
+export interface IRulesCache {
+  [key: string]: RuleItemInfoBase[] | [];
```

Comment:
Don't need to specify `| []` separately, an array can be empty for the specified type